### DISCUSSION TITLE
hello!

### DISCUSSION THREAD

**Discussion by Phisbin** [ZuluSCSI/ZuluSCSI-firmware#56] [Q&A] [answered]
I am replacing a rear admiral thunderdrive scsi hard disk that just went bad....it is a CMD HD clone  and  it is limited to a 4.1gb disk max.  how would I go about making a image drive for this commodore device?   it needs to be 4.1 gb, and I have to use CMD's hd toolkit disk that came with the drive to low lvl format and partition the drive.  I was running this as device 9, and my 1581 is device 8 atm....drive 12 is a sd2iec......do I just put a 8 gb sd card I to the thunderdrive with zuluscsi running and run my setup disk in my physical drive 8  to format this.....if I first have to put a image what would it be on the SD card.....hd0.hda? any help would be appreciated....need my BBS back up and running

**Comment by aperezbios** [comment, marked as answer]
All you have to do is create an image file (it can be empty, or contain transplanted data), that's 4GB or smaller, on your SD card What you name the image file (HD0.img, HD0.hda, etc) doesn't particularly matter; Only that the file name starts with HD, and has a unique, valid SCSI ID of 0-7). From that point on, you need to treat the ZuluSCSI as you would any new, blank hard drive, from the perspective of the computer it's attached to.

You can create empty files any number of ways (using dd at a command prompt, or various GUI utilities). If you're using windows, I can recommend http://www.mynikko.com/dummy/
